src/app/routes/edges/routes/PrivateEdgeList/components/SlatList/components/ClusterStatusInfo.js

```diff
@@ -61,8 +89,15 @@ function ClusterStatusInfo({
                   }}
                 >
                   Last check in&nbsp;
-                  {cluster.metadata.modifiedAt ? (
-                    <span>{Moment(cluster.metadata.modifiedAt).fromNow()}</span>
+                  {lastCheckedIn != null ? (
+                    <span>
```

Comment:
We have [DateFormat](https://github.com/RafayLabs/rcloud-console-ui/blob/main/src/components/DateFormat.js) if you want to switch to that.

Reply:
what we display here is not the DateTime similar to project creation. It's like this "Last check-in  6 days ago" and for that we need to use Moment.